### ISSUE TITLE
Commands do not work with Node v20.6: `Cannot redefine property: __internal__deprecationWarning`

### ISSUE THREAD

**Issue by AlexLavoie42** [nuxt/nuxt#23020]
### Environment

------------------------------
- Operating System: Linux
- Node Version:     v20.6.0
- Nuxt Version:     3.7.1
- CLI Version:      3.7.3
- Nitro Version:    2.6.2
- Package Manager:  pnpm@8.7.3
- Builder:          -
- User Config:      extends, nitro, css, app, imports, modules, fontLoader, mapbox, quasar
- Runtime Modules:  nuxt-mapbox@1.4.4, @nuxtjs/device@3.1.0, nuxt-font-loader@2.3.1
- Build Modules:    -
------------------------------

### Reproduction

After upgrading to node v20.6, you should get `Cannot redefine property: __internal__deprecationWarning` after trying to run `nuxi build` or some other commands as well.

### Describe the bug

I am fairly certain this is related to [this issue](https://github.com/nodejs/node/issues/49497). Don't think there is anything that can be done currently but I wanted to open an issue in case others run into the problem. This can be easily worked around by downgrading your node version to something other than v20.6.

### Logs

```shell-script
> nuxt build

Nuxt 3.7.1 with Nitro 2.6.2                                                                                                                                                                                  4:48:24 PM

 ERROR  Cannot redefine property: __internal__deprecationWarning                                                                                                                                             4:48:24 PM

  at Function.defineProperty (<anonymous>)
  at Object.<anonymous> (/home/alex/ProperateMono/node_modules/.pnpm/@babel+types@7.22.15/node_modules/@babel/types/lib/index.js:66:8)
  at Module._compile (node:internal/modules/cjs/loader:1241:14)
  at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
  at Module.load (node:internal/modules/cjs/loader:1091:32)
  at Module._load (node:internal/modules/cjs/loader:938:12)
  at Module.require (node:internal/modules/cjs/loader:1115:19)
  at require (node:internal/modules/helpers:130:18)
  at Object.<anonymous> (/home/alex/ProperateMono/node_modules/.pnpm/@babel+types@7.22.15/node_modules/@babel/types/lib/builders/validateNode.js:8:14)
  at Module._compile (node:internal/modules/cjs/loader:1241:14) 



 ERROR  Cannot redefine property: __internal__deprecationWarning                                                                                                                                             4:48:24 PM

 ELIFECYCLE  Command failed with exit code 1.
```

**Comment by kingychiu** [comment]
Thanks! I also saw this on CI today; I pin the node version for the build layer in the docker image for now.
```
FROM node:20.5.1
```

**Comment by danielroe** [comment]
cc: @pi0

**Comment by mklueh** [comment]
Same. 

v20.4.0 works. Moments when you really appreciate NVM :)

**Comment by catsmeatman** [comment]
I see this bug too.

Operating System: MacOS Ventura 13.5.1
Node Version: v20.6.0
Nuxt Version: 3.7.1
CLI Version: 3.7.3
Nitro Version: 2.6.2

**Comment by binochoi** [comment]
i had to use previous version that is 20.5.0

**Comment by nicolo-ribaudo** [comment]
We added a workaround for this in `@babel/core`, `@babel/traverse` and `@babel/types` 7.22.17

**Comment by steklopod** [comment]
Node version update in my github action solved the error (latest nuxt 3.7.1):
<img width="1063" alt="Снимок экрана 2023-09-09 в 15 07 22" src="https://github.com/nuxt/nuxt/assets/29500153/51637dc1-cab5-45cd-9e2c-8e17e00d696c">

**Comment by so1ve** [comment]
nodejs/help#4241

**Comment by VividLemon** [comment]
Is this the reason why my CI isn't working? https://github.com/bootstrap-vue-next/bootstrap-vue-next/actions/runs/6137959608/job/16654341216

**Comment by steklopod** [comment]
> Is this the reason why my CI isn't working? https://github.com/bootstrap-vue-next/bootstrap-vue-next/actions/runs/6137959608/job/16654341216

You have 
```yml
  - name: Install Node.js
        uses: actions/setup-node@v3
        with:
          node-version: 20.x
```

Try ` node-version: latest`

**Comment by VividLemon** [comment]
> You have
> 
> ```yaml
>   - name: Install Node.js
>         uses: actions/setup-node@v3
>         with:
>           node-version: 20.x
> ```
> 
> Try ` node-version: latest`

So `20.x` syntax doesn't do patch versions? Seems strange.

**Comment by craigvantonder** [comment]
Same issue for me with 20.6.0 when running npm install after upgrading nuxt:
nuxt        ^3.6.5  →   ^3.7.3
Can confirm that downgrading node to 20.5.0 and running `npm install` produces no errors as was indicated in https://github.com/nuxt/nuxt/issues/23020#issuecomment-1708101988.

**Comment by nicolo-ribaudo** [comment]
@craigvantonder The Node.js bug has been fixed in version 20.6.1: you can upgrade instead of downgrading :)

**Comment by craigvantonder** [comment]
@nicolo-ribaudo These are both good points, thank you :-)
